spec/requests/people_n_plus_one_spec.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe "n+1 queries in the people controller" do
+  describe "#search", n_plus_one: true do
```

Comment:
This is testing `#index`, not `#search`. It would be nice to have both.

And then search needs the `with_attached_*` here: https://github.com/otwcode/otwarchive/blob/1588f6c552b1c85adeb1b30e31411cb2cb6f37d7/app/controllers/people_controller.rb#L11

(Not quite sure about the exact line number.)